redgenes/gene_annotations.py

```diff
@@ -163,26 +262,72 @@ def annotation_pipeline(df):
         )
 
         ## Step 4: Run barrnap to get {filename}_rrna.gff
-        run_barrnap(unzipped_input, outdir, filename_stem)
+        run_barrnap(unzipped_input, outdir, filename_stem, True)
 
         ## Step 5: Zip fna file
         run_zip_fna(unzipped_input)
 
         ## Step 6: Insert results into tables
         ## prodigal
-        cds_results = extract_gff_contents(f"{Path(outdir) / f'{filename_stem}.gff'}")
+        cds_results = extract_prodigal_results(
+            f"{Path(outdir) / f'{filename_stem}_cds.gff'}"
+        )
+
+        ## kofamscan
+        ko_results = extract_kofamscan_results(
+            f"{Path(outdir) / f'{filename_stem}_kofamscan.tsv'}"
+        )
+
+        ## barrnap
+        rrna_results = extract_barrnap_results(
+            f"{Path(outdir) / f'{filename_stem}_rrna.gff'}"
+        )
 
         ## identifier and md_info
-        sql_identifier = "insert into identifier (filename_full, filepath) values (?, ?) returning genome_id"
+        sql_identifier = "insert into identifier (filename_full, filepath) values (?, ?) returning entity_id"
```

Comment:
A reorg like the below can help with readability 

```suggestion
        sql_identifier = """INSERT INTO identifier (filename_full, filepath) 
                            VALUES (?, ?) 
                            RETURNING entity_id"""
```